ow_lander/config/joint_limits.yaml

```diff
@@ -46,4 +46,7 @@ joint_limits:
     has_velocity_limits: true
     max_velocity: 0.15
     has_acceleration_limits: false
-    max_acceleration: 0
\ No newline at end of file
+    max_acceleration: 0
+
+default_acceleration_scaling_factor: 1.0
```

Comment:
In noetic-devel I added these lines indented just underneath "joint_limits:" I don't know if their indentation or position in the file makes any difference. Is this the best place to put them?

Reply:
Yes, if you indent them they would not be configured correctly.

Reply:
Okay. And that's odd, since they fixed the problem in noetic even though I had them in the wrong place.

Reply:
It is confusing but I think both melodic and noetic expect them without indention. Otherwise, how do you explain the slow movement in the noetic branch even though you had them defined.

Reply:
The lines I added fixed noetic-devel at the time. We tested it and it worked. If the position of those lines no longer work for noetic-devel then it probably means there was a modification to the way the file is parsed.